lib/cc/yaml/nodes/glob_list.rb

```diff
@@ -3,6 +3,10 @@ module Yaml
     module Nodes
       class GlobList < Sequence
         type Glob
+
+        def allow_child?(child)
```

Comment:
I realized write after I wrote this that wanting to quell the immediate fire was your main priority & evaluating the unknown risk was not important to do now.

Your current reasoning makes sense to me. Carry on.